src/components/CreateEmployeeForm.tsx

```diff
@@ -1,128 +1,184 @@
-import React, { FC } from 'react';
+import React, { FC, useState } from 'react';
 import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
 import { yupResolver } from '@hookform/resolvers/yup';
 import * as yup from 'yup';
 
 import DropdownMenu from './DropdownMenu';
 import InputField from './InputField';
 import Label from './Label';
 import Button from './Button';
+import FileInput from './FileInput';
+import { useAddEmployeeMutation, useAddFileMutation, useGetDepartmentListQuery, useGetRoleListQuery }
+    from 'services/api';
+import PopUp from './PopUp';
 
 const schema = yup.object({
```

Comment:
move the validations to a file within the corresponding container. Keep the file name as validate.ts. You can import the same form there and use in the respective components.